pkg/client/docker.go

```diff
@@ -21,6 +21,7 @@ type DockerClient interface {
 	ImageRemove(ctx context.Context, image string, options types.ImageRemoveOptions) ([]types.ImageDeleteResponseItem, error)
 	ImagePull(ctx context.Context, ref string, options types.ImagePullOptions) (io.ReadCloser, error)
 	Info(ctx context.Context) (types.Info, error)
+	ServerVersion(ctx context.Context) (types.Version, error)
```

Comment:
Do we also need this in `pkg/testmocks/mock_docker_client.go`?

Reply:
I just noticed that it is already there.

https://github.com/buildpacks/pack/blob/cf17f4c8263de88a5b6646acba5d03f457919fa9/pkg/testmocks/mock_docker_client.go#L1429-L1436

Reply:
Nice! I suppose that is why the tests passed. Let me know if anything else is needed, and thanks for the review and comments.